scss/vendor/_rfs.scss

```diff
@@ -2,38 +2,41 @@
 
 // SCSS RFS mixin
 //
-// Automated responsive font sizes
+// Automated responsive values for for font sizes, paddings, margins and much more
```

Comment:
double for

Reply:
Upstream PR: https://github.com/twbs/rfs/pull/161